README.md

```diff
@@ -1,8 +1,11 @@
 # Boots
 [![Build Status](https://cloud.drone.io/api/badges/tinkerbell/boots/status.svg)](https://cloud.drone.io/tinkerbell/boots)
+![](https://img.shields.io/badge/Stability-Maintained-green.svg)
 
 This services handles DHCP, PXE, tftp, and iPXE for provisions.
 
+This repository is [Maintained](https://github.com/packethost/standards/blob/master/maintained-statement.md) meaning that this software is supported by Tinkerbell and its community - available to use in production environments.
```

Comment:
> available to use in production environments

I'm worried this signals something that is not true.

Reply:
Which part are you concerned about? The Tinkerbell bit or the supported bit or production environment bit or ...?